contractcourt/htlc_timeout_resolver.go

```diff
@@ -888,13 +962,20 @@ func (h *htlcTimeoutResolver) initReport() {
 		)
 	}
 
+	// If there's no timeout transaction, then we're already effectively in
+	// level two.
+	stage := uint32(1)
```

Comment:
Since there's no need to do two-level sweeping I think the direct spend should be stage one?

Reply:
In the UTXO nursery, direct spends are currently considered stage 2.  We should probably keep this behavior for consistency.

Reply:
Yeah this is a sorta technicality. Initially the itest was failing as it expected stage 2, but was given a stage 1 output. Could've modified the itest, but instead opted to change this.